sonar-ws/src/main/protobuf/ws-issues.proto

```diff
@@ -65,7 +65,7 @@ message Issue {
   optional string rule = 2;
   optional sonarqube.ws.commons.Severity severity = 3;
   optional string component = 4;
-  optional int64 componentId = 5;
+  optional int64 deprecatedComponentId = 5;
```

Comment:
Renaming this parameter was a good idea, but the name should start with "unused"